CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.6.3 -- 2020-05-08
```

Comment:
Please don't add this line, and keep this change in the `Unreleased` section, I add the line when I actually do the release. :)

Reply:
Of course! :)